cdci_data_analysis/flask_app/app.py

```diff
@@ -151,9 +151,14 @@ def meta_data_src():
 @app.route("/download_products", methods=['POST', 'GET'])
 def download_products():
     query = InstrumentQueryBackEnd(app, download_products=True)
-    return query.download_products()
+    return query.download_file()
 
 
+@app.route("/download_file", methods=['POST', 'GET'])
```

Comment:
could you please explain why this is needed?

Reply:
is your comment about code duplication with the `download_products` endpoint ?

Reply:
the behavior is slightly different, and mainly not to use the `download_products` endpoint when the intention is to download a single file (from the dedicated folder) instead of the the whole products for a given job (from the scratch_dir)

Reply:
it redirects now, adding specific args